neqo-transport/src/recovery.rs

```diff
@@ -211,12 +228,102 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
```

Comment:
This should probably go in a new file.  There's going to be more here, right?

Reply:
maybe. It's not clear that congestion control is separable from recovery internals...